input/parser.go

```diff
@@ -1,72 +1,98 @@
 package input
 
 import (
-	"regexp"
-	"strconv"
+	"encoding/json"
+	"strings"
+	"time"
+
+	"github.com/jeromer/syslogparser"
```

Comment:
Unfortunately I don't want anymore 3rd party libraries in the source. I want to minimize their use.

https://blog.gopheracademy.com/advent-2014/case-against-3pl/

I would be up for copying and pasting the code over however, with full attribution. I do think moving away from regex is the right thing to do.